Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -0,0 +1,78 @@
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Doppler.HtmlEditorApi.DopplerSecurity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Doppler.HtmlEditorApi.Controllers
+{
+    [Authorize]
+    [ApiController]
+    public class CampaignsController : Controller
+    {
+        [HttpGet("/accounts/{accountName}/campaigns/{campaignId}/content/design")]
+        public string GetCampaign(string accountName, int campaignId)
```

Comment:
The problem with this approach is the content type. It should be JSON and it is string (text or something like that).

What do you think about defining a class for this model? the class will be useful in the next steps.

Reply:
Fixed